src/Command.php

```diff
@@ -172,7 +172,7 @@ public function setCommand($command)
     }
 
     /**
-     * @param string $stdIn If set, the string will be piped to the command via standard input.
+     * @param string|resource $stdIn If set, the string will be piped to the command via standard input.
```

Comment:
@Arzaroth Maybe add a little note about the resource type here? E.g. "This can also be a resource like a file handle in which case the file content will be piped into the command."

Reply:
Sure thing.